lib/jsonapi_parameters/default_handlers/base_handler.rb

```diff
@@ -19,6 +25,10 @@ def find_included_object(related_id:, related_type:)
                 included_object_enum[:type] == related_type
             end
           end
+
+          def self.call(key, val, included)
+            new(key, val, included).handle
+          end
```

Comment:
In Ruby any object that responds to `call` can be used in place of labda or proc. This simplify default handlers.